src/libraries/Microsoft.Win32.Primitives/src/Microsoft.Win32.Primitives.csproj

```diff
@@ -14,7 +14,7 @@
              Link="Common\Interop\Windows\Interop.FormatMessage.cs" />
     <Compile Include="System\ComponentModel\Win32Exception.Windows.cs" />
   </ItemGroup>
-  <ItemGroup Condition=" '$(TargetsUnix)' == 'true'">
+  <ItemGroup Condition=" '$(TargetsUnixOrBrowser)' == 'true'">
```

Comment:
I don't think this will work

Reply:
Was discussing this one with @Anipik.  This contains only an exception.  We shouldn't have Win32Exception throw PlatformNotSupportedException.  We should probably have a build and just default out the one method that touches the OS